### PR TITLE
[12.0][IMP] stock_mts_mto_rule: Improve inheritance

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT30217

please @sergio-teruel @chienandalu review it

### CODE REVIEW

File: stock_mts_mto_rule/models/stock_rule.py [hunk 1]
```diff
@@ -30,16 +30,19 @@ def _check_mts_mto_rule(self):
...
                     raise ValidationError(msg)
 
+    def _get_qty_available_for_mto_qty(self, product, product_uom):
+        src_location_id = self.mts_rule_id.location_src_id.id
+        product_location = product.with_context(location=src_location_id)
+        virtual_available = product_location.virtual_available
+        return product.uom_id._compute_quantity(virtual_available, product_uom)
+
     @api.multi
-    def get_mto_qty_to_order(self, product, product_qty, product_uom, values):
```
Comment (chienandalu) [inline stock_mts_mto_rule/models/stock_rule.py:34]: Maybe you're potentially breaking methods that inherit this one. Is it needed to remove this parameter?

Comment (CarlosRoca13) [reply stock_mts_mto_rule/models/stock_rule.py:34]: Ups, true sorry, I recover this changes

**Review by chienandalu — COMMENTED** [review]
Apart from adding a method, this doesn't change the logic. Isn't it?

File: stock_mts_mto_rule/README.rst [hunk 2]
```diff
@@ -45,49 +69,46 @@ Usage
...
 You should not select both the mts+mto route and the mto route.
 
-.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
-   :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/153/12.0
-
-Known issues
-============
-
-If you cancel a delivery order and then recreate it from Recreate
```
Comment (chienandalu) [inline stock_mts_mto_rule/README.rst:55]: Do this apply or not? Maybe you forgot to add the proper README part?

**Comment by CarlosRoca13** [comment]
@chienandalu 
> Apart from adding a method, this doesn't change the logic. Isn't it?

Yes, it is. It doesn't change the logic

And changes done

**Review by chienandalu — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
This should be done in upper versions as well.

**Comment by CarlosRoca13** [comment]
Added change that @sergio-teruel commented me in private

**Pull request merged**